src/main/java/Main.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Arrays;
+
+public class Main {
+    public static void main (String[] args){
+
+        Ball ball1 = Lottery.getRandomBall();
+        Ball ball2 = Lottery.getRandomBall();
+        Ball ball3 = Lottery.getRandomBall();
+
+        System.out.println(ball1);
+        System.out.println(ball2);
+        System.out.println(ball3);

```

Comment:
Please use cycle for repetitive actions

Reply:
if i did all changes, i must to do a new pull request?